tests/main/test_home_page.py

```diff
@@ -13,6 +13,12 @@ def test_not_staff_user_redirected_from_admin_login(self, client, persist_user):
         assert response.status_code == 302
         assert response.url == reverse('homePage')
 
+    def test_successful_logout(self, client, persist_user):
```

Comment:
We should be consistent here. login/logout or sign in/sign out

Reply:
Yes, I fixed it to login/logout.